### PR TITLE
Update sitemap API to get pages dynamically from folder

### PR DESCRIPTION
**Pull request by cp-jagruti-a**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [ ] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [ ] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [ ] Make MR mark as ready before submitting it for review
- [ ] Add images, videos, or gifs of UI
- [ ] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

### CODE REVIEW

File: sitemap/sitemap.go [hunk 1]
```diff
@@ -72,25 +74,28 @@ type URLset struct {
...
+		{Loc: baseUrl + "/resources", Priority: `0.9`},
 	}
 
-	// add portfolios
-	sitemapUrls = addPortfolios(baseUrl, sitemapUrls)
-
-	// add all jobs
-	sitemapUrls, err := repository.addJobs(baseUrl, sitemapUrls)
+	var folderPath string
+	if testFolderPath != "" {
```
Comment (cp-sumi-k) [inline sitemap/sitemap.go:88]: Where `testFolderPath` is setting?

File: sitemap/sitemap.go [hunk 2]
```diff
@@ -72,25 +74,28 @@ type URLset struct {
...
+		folderPath = testFolderPath
+	} else {
+		folderPath = "nuxt-frontend/pages"
+	}
 
+	//add unique services
```
Comment (cp-sumi-k) [inline sitemap/sitemap.go:94]: Remove this comment if not required

File: sitemap/sitemap.go [hunk 3]
```diff
@@ -72,25 +74,28 @@ type URLset struct {
...
+	sitemapUrls, err := repository.addPages(baseUrl, sitemapUrls, folderPath)
 	if err != nil {
 		log.Error(err)
-		c.AbortWithStatus(http.StatusInternalServerError)
-		return
+		// Handle error
```
Comment (cp-sumi-k) [inline sitemap/sitemap.go:98]: Why `aborting` is removed?

File: nuxt-frontend/pages/unsubscribe.vue
Comment (cp-sumi-k) [inline nuxt-frontend/pages/unsubscribe.vue:1]: <UnSubscribe /> is component of canopas-blog-components. and its working code. Let's not remove it

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: sitemap/sitemap.go (hunk 1 above)
Comment (cp-jagruti-a) [reply sitemap/sitemap.go:88]: its name is relevant as its used in test file and set there

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**